packages/jest-haste-map/src/types.js

```diff
@@ -10,6 +10,7 @@
 'use strict';
 
 import type {Error} from 'types/TestResult';
+import type {InternalHasteMap} from 'types/HasteMap';
 import type {ModuleMetaData} from 'types/HasteMap';
```

Comment:
Can you merge these two imports? You are importing the two types from the same file.